client/my-sites/theme/main.jsx

```diff
@@ -555,7 +556,7 @@ class ThemeSheet extends React.Component {
 						</div>
 					</div>
 					<Button primary={ true } href={ '/themes/' }>
-						{ i18n.translate( 'See All Themes' ) }
+						{ i18n.translate( 'See all themes' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.